sites/public/src/ResourceLinkCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { LinkButton, Icon } from "@bloom-housing/ui-components"
+import { IconTypes } from "../../../ui-components/src/icons/Icon"
+
+interface ResourceLinkCardProps {
+  iconSymbol: IconTypes
+  title: string
+  subtitle: string
+  linkUrl: string
+  linkLabel: string
+}
+
+const ResourceLinkCard = (props: ResourceLinkCardProps) => {
+  const { iconSymbol, title, subtitle, linkUrl, linkLabel } = props
+
+  return (
+    <div className="border-b">
+      <div className="my-6 md:my-12">
+        <Icon
```

Comment:
`<Icon symbol="house" size="2xl" className="border-primary border-b-4 inline-block pb-3 px-3 mb-2" />`
Here is what the Icon looks like from the comparable one in Alameda